src/main/java/org/broadinstitute/consent/http/models/DacDecisionMetrics.java

```diff
@@ -181,15 +181,19 @@ private void setAverageTurnaroundTimeMillis(Double averageTurnaroundTimeMillis)
     this.averageTurnaroundTimeMillis = averageTurnaroundTimeMillis;
   }
 
-  public String getAverageTurnaroundTime() {
+  public Integer getAverageTurnaroundTime() {
     return averageTurnaroundTime;
   }
 
   private void setAverageTurnaroundTime() {
     if (Objects.nonNull(this.getAverageTurnaroundTimeMillis())) {
-      this.averageTurnaroundTime =
-          DurationFormatUtils.formatDurationWords(
-              this.getAverageTurnaroundTimeMillis().longValue(), true, true);
+      //this will only ever catch an exception if there is no final date, or an unreasonable amount of time between them
+      //in this case, the upward bound of Integer is displayed
+      try {
+        this.averageTurnaroundTime = Math.toIntExact(TimeUnit.MILLISECONDS.toDays(this.averageTurnaroundTimeMillis.longValue()));
+      } catch (ArithmeticException e) {
+        this.averageTurnaroundTime = 2147483647;
+      }
```

Comment:
okay good idea!! and yes we should be able to convert any long to an integer but since the method Math.toIntExact throws an exception if the long is too large to be converted to an integer I thought it would be safer to wrap it just in case